Adding end of life notice for Sufia

jeremyf opened the pull request:
Based on conversations on Slack and through email, the community has
agreed that Sufia is end-of-life, but that decision may not be as
obvious to those outside the community's communication channels.

jeremyf committed:
Adding end of life notice for Sufia

Based on conversations on Slack and through email, the community has
agreed that Sufia is end-of-life, but that decision may not be as
obvious to those outside the community's communication channels.

jcoyne commented:
Can we bring this up on the tech call before merging?

jeremyf commented:
@jcoyne Sounds like a good plan.

mjgiarlo reviewed (changes requested):
Blocking merger of this PR 'til we discuss on Wednesday's tech call.

mjgiarlo commented on README.md:
```diff
@@ -15,6 +15,16 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
 Jump in: [![Slack Status](http://slack.samvera.org/badge.svg)](http://slack.samvera.org/)
 [![Ready Tickets](https://badge.waffle.io/samvera/sufia.png?label=ready&title=Ready)](https://waffle.io/samvera/sufia)
 
+# End of Life Notice
```
I wonder if we should replace pretty much everything in this README with this section so folks don't miss it.

mjgiarlo commented on README.md:
```diff
@@ -15,6 +15,16 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
+We envision the 7.x releases as the end-of-life for Sufia. During 2017, the [Samvera community](http://samvera.org/) consolidated Sufia and [CurationConcerns](https://github.com/samvera/curation_concerns) into [Hyrax](https://github.com/samvera/hyrax):
+
+* [planning notes on consolidation](https://wiki.duraspace.org/pages/viewpage.action?pageId=78161232))
+* [release notes on the bridge of Sufia to Hyrax](https://github.com/samvera/hyrax/releases/tag/v1.0.0.rc2)
```
Change to https://github.com/samvera/hyrax/releases/tag/v1.0.1

mjgiarlo replied:
Can you also add a line below this one:

```markdown
* [release notes on the bridge of CurationConcerns to Hyrax](https://github.com/samvera/hyrax/releases/tag/v2.0.0)
```
❓

jcoyne commented:
Please ask the tech list if anyone objects to us considering  CC & Sufia EOL.  if no objections, then we can merge this.

jeremyf committed:
Adding additional links for end of life

Also pushing the table of contents further down to better draw
attention to the end of life.

jeremyf commented:
https://groups.google.com/forum/#!topic/samvera-tech/guEUaByL8Y0

jeremyf committed:
Amending language to indicate shift in focus

Based on the following discussions, reworking the language from end-of-
life to shift in focus.

* https://groups.google.com/forum/#!topic/samvera-tech/guEUaByL8Y0
* https://groups.google.com/forum/#!topic/samvera-community/VH4RB6WSFeE

mjgiarlo commented on README.md:
```diff
@@ -15,6 +15,29 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
+During 2017, the [Samvera community](http://samvera.org/) consolidated Sufia and [CurationConcerns](https://github.com/samvera/curation_concerns) into [Hyrax](https://github.com/samvera/hyrax):
+
+* [planning notes on consolidation](https://wiki.duraspace.org/pages/viewpage.action?pageId=78161232))
+* [release notes on the bridge of Sufia to Hyrax](https://github.com/samvera/hyrax/releases/tag/v1.0.0)
```
Can you change this to point at 1.0.1? We had to yank 1.0.0 because it was released from the wrong branch.

mjgiarlo commented on README.md:
```diff
@@ -15,6 +15,29 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
+
+During 2017, the [Samvera community](http://samvera.org/) consolidated Sufia and [CurationConcerns](https://github.com/samvera/curation_concerns) into [Hyrax](https://github.com/samvera/hyrax):
+
+* [planning notes on consolidation](https://wiki.duraspace.org/pages/viewpage.action?pageId=78161232))
```
This is on our old wiki. Can you link to https://wiki.duraspace.org/pages/viewpage.action?pageId=87461232 instead?

mjgiarlo reviewed (changes requested):
Looking good. Just a couple links to tweak. Thanks!

mjgiarlo commented:
@jeremyf Is this PR still relevant and worth updating?
